tests/rbd_mirror/test-image-delete-from-secondary.py

```diff
@@ -0,0 +1,100 @@
+import datetime
+import time
+
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """verification of removal of failed back images should reflect to secondary.
+
+    this verifies that the deletion of primary images deletes the images on secondary also when we perform multiple
+    promote and demote operation.
+
+    Args:
+        **kw:
+        repeat_count - number of times promote and demote happens before delete image
+    Returns:
+        0 - if test case pass
+        1 - it test case fails
+
+    Test case flow:
+    1. Create image on cluster-1.
+    2. Enable mirroring for created image on cluster-1 and run IOs.
+    3. Ensure mirror image created on cluster-2.
+    4. Force promote image on cluster-2 and run IOs.
+    5. demote image on cluster 1.
+    6. resync the image to cluster 2.
+    7. Demote image on cluster-2 and promote image on cluster-1.
+    8. Delete image on cluster-1.
+    """
+    try:
+        log.info("Starting RBD mirroring test case - CEPH-83574741")
+        config = kw.get("config")
+        mirror1, mirror2 = [
+            rbdmirror.RbdMirror(cluster, config)
+            for cluster in kw.get("ceph_cluster_dict").values()
+        ]
+
+        # create images and enable snapshot mirroring
+        poolname = mirror1.random_string() + "83574741pool"
```

Comment:
Can we induce options of taking pool ame, image name from suites ? so that we can reuse sae pool and images in different TCs ?

Reply:
as discussed we are keeping it as it and follow to provide pool and image name from suite